spec/integration/apply_spec.rb

```diff
@@ -105,6 +105,18 @@ def agent_version_inventory
         end
       end
 
+      it 'gets resources' do
+        with_tempfile_containing('inventory', YAML.dump(agent_version_inventory), '.yaml') do |inv|
+          results = run_cli_json(%W[plan run basic::resources --nodes agent_targets
+                                    --modulepath #{modulepath} --inventoryfile #{inv.path}])
+          results.each do |result|
+            expect(result['status']).to eq('success')
+            resources = result['result']['resources']
+            expect(resources.map { |r| r['type'] }.uniq).to eq(%w[User File])
```

Comment:
Should we make sure File['/tmp'] is returned and is the only File resource and that at least the bolt user is returned?

Reply:
Done.